openassessment/xblock/static/js/src/lms/oa_base.js

```diff
@@ -68,6 +69,95 @@ OpenAssessment.BaseView.prototype = {
     },
 
     /**
+     * Add the text messages to the Aria live region (first creating it if necessary).
+     *
+     * @param {string[]} texts
+     */
+    srReadTexts: function(texts) {
+        var readerFeedbackID = '.sr.reader-feedback',
+            $readerFeedbackSelector = $(readerFeedbackID),
+            htmlFeedback = '';
+
+        if ($readerFeedbackSelector.length === 0) {
```

Comment:
Actually, instead of appending dynamically why can't this just always exist (via a template)? I'm concerned that text may not be read initially when it is dynamically injected-- @cptvitamin should know whether or not that is a problem.

But also for simplicity, it seems that it would be better if this region is just always there. I think you could put it in oa_base.html.